client/spring-zeebe-starter/src/main/java/io/camunda/zeebe/spring/client/properties/ZeebeClientConfigurationProperties.java

```diff
@@ -586,4 +599,9 @@ public JsonMapper getJsonMapper() {
     return jsonMapper;
   }
 
+  @Override
+  public Map<String, WorkerConfiguration> getWorkersConfiguration() {
```

Comment:
Why do we name it differently in the interface than in the config-object (=properties/yaml) file? I think it should be the same to avoid any confusion.

If we want to separate the usage from the definition, I would probably see 

`getWorkerConfiguration(String workertype)`

as a useful interface/abstraction - WDYT?

Then this would probably hint towards using `zeebe.client.worker.config['foo'].enabled=false` as a property name?
Or would `zeebe.client.worker.foo.enabled=false` work somehow? Not sure about Spring here - do you know? 

I wonder if we make any checks if people define multiple workers for one type 🤔

Reply:
Right now there is no direct dependency and we can use the map directly from the `Worker` class :)